ThemeDecorator/AccessibilityDecorator.js

```diff
@@ -14,125 +14,124 @@ import {Component} from 'react';
  * @public
  */
 const AccessibilityDecorator = hoc((config, Wrapped) => {
-	return class extends Component {
-		static contextType = ResizeContext;
+	const Accessibility = (props) => {
+		const {className, focusRing, highContrast, skinVariants, textSize = 'normal', ...rest} = props;
```

Comment:
why didn't you use setDefaultProps function from enact/core/util?

Reply:
I thought that it would be unnecessary to use the `setDefaultProps` function since the component has just one property with a default value

Reply:
ok, let's use it to align with other components